OnlineShoppingXamarin/OnlineShoppingXamarin/Model/Cart.cs

```diff
@@ -6,10 +6,10 @@ namespace OnlineShoppingXamarin.Model
 {
     public class Cart
     {
-        public Guid CartId { get; set; }
-        public Guid UserId { get; set; }
+        public int CartId { get; set; }
```

Comment:
Why did we changed all the Ids to be int instead of GUID ??

Reply:
3lashan bs fel dummy data lama byndh ll service kol mara by generate guid gded f sabto int bs 3lashan fel dummy data yfdl sabet bs fel api zy ma howa 3ady guid